### ISSUE TITLE
Support for inline field validation message?

### ISSUE THREAD

**Issue by kctang** [studiointeract/accounts-ui#49]
Customizing a theme based on this package...progress is good but can't seem to get the form error messages to show next to input fields.

I am using [MDL](https://getmdl.io/components/index.html#textfields-section). It looks something like:

``` html
  <div class="mdl-textfield mdl-js-textfield">
    <input class="mdl-textfield__input" type="text" id="sample2">
    <label class="mdl-textfield__label" for="sample2">Number...</label>
    <!-- render this on error -->
    <span class="mdl-textfield__error">Input is not a number!</span>
  </div>
```

Any hints on how this can be done or is this a feature request?

**Comment by timbrandin** [comment]
Great suggestion, have myself thought about adding this. Though my time is limited, so this would likely take some time before I can look into, if you find the time to write a PR, I'd be happy to review and merge it!

**Comment by kctang** [comment]
Ok good to know that you have this in mind. As much as I would love to, I don't think I can provide a PR anytime soon. I'll take a look after my current project is done if others didn't get to it first.

Thanks!

**Comment by timbrandin** [comment]
@kctang I have built a solution for this on the devel branch now, would be great to get your oppinion on it.

**Comment by kctang** [comment]
Sorry but i can't try it out now...busy with other project for the next couple of weeks. :-(

**Comment by timbrandin** [comment]
This is published now in `v1.2.12`. :)
